tests/integration_tests/build/test_binary_static_linking.py

```diff
@@ -8,15 +8,15 @@
 import host_tools.cargo_build as host
 from framework import utils
 
+
```

Comment:
changes in this file don't seem to belong to the commit. Can be a separate commit?

Reply:
fixed